packages/core/src/Modules/Cashier/Components/Email/send-email.jsx

```diff
@@ -18,9 +18,12 @@ class SendEmail extends React.Component {
                 ) : (
                     <React.Fragment>
                         <Icon icon='IcCashierAuthenticate' className='withdraw__icon' size={128} />
-                        <p className='withdraw__header'>
+                        {/* <p className='withdraw__header'>
                             <Localize i18n_default_text='To initiate a withdrawal, we need to authenticate you via email.' />
-                        </p>
+                        </p> */}
```

Comment:
Can we refactor the commented code as well? In case of saving time and prevent human mistake when someone is going to bring this back.

Reply:
This card is only to reach an agreement about the structure. As Zaki said, we would create a separate card to refactor all components throughout the project.